libraries/botbuilder/src/teamsInfo.ts

```diff
@@ -58,6 +86,27 @@ export class TeamsInfo {
         return this.getMembersInternal(this.getConnectorClient(context), t);

```

Comment:
>this [](start = 15, length = 4)

missing "await"

Reply:
The other option is to just drop the "async" at start of this function

---
In reply to: [387314847](https://github.com/microsoft/botbuilder-js/pull/1833#discussion_r387314847) [](ancestors = 387314847)